test_files/export_multi/export_multi.ts

```diff
@@ -15,7 +15,15 @@ export const {
   ORANGE,
 } = Fruit;
 
-export const {a, b} = {
+export const {a, b, c, d, e, f, g, h, i, j} = {
```

Comment:
this test was here (I think) to ensure the intializer on the right hand side doesn't get repeated. Maybe add a separate export statement for the "10 members" case?

Reply:
OK, sure.  It doesn't hurt to have more cases tested.  Added separate 2 element and 10 element cases.

Reply:
Added both 2-element and 10-element cases.